src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.VirtualMonoRepo.E2E.Tests/Microsoft.DotNet.Darc.VirtualMonoRepo.E2E.Tests.csproj

```diff
@@ -47,4 +47,8 @@
     </None>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Drawing.Common" />
```

Comment:
Is this intentional?

Reply:
Yeah, it won't build on my computer without it. That's where "Bitmap" is, which these tests depend on.